detour_test.go

```diff
@@ -69,30 +70,6 @@ func TestBlockedImmediately(t *testing.T) {
 	}
 }
 
-func TestBlockedAfterwards(t *testing.T) {
```

Comment:
This test no longer works because we're explicitly bypassing the detour logic after a successful connection. This is analogous to what we used to do with a PAC file.